clouddriver-web/src/main/groovy/com/netflix/spinnaker/clouddriver/controllers/ReservationReportController.groovy

```diff
@@ -31,6 +32,11 @@ class ReservationReportController {
 
   @RequestMapping(method = RequestMethod.GET)
   Collection<ReservationReport> getReservationReports() {
-    reservationProviders.collect { it.reservationReport } - null
+    reservationProviders.collect { it.getReservationReport("v1") } - null
```

Comment:
Defaults to the legacy report that does not break out vpc reservations.

`deck` will continue using this endpoint until it's updated.